### ISSUE TITLE
Suggestion

### ISSUE THREAD

**Issue by vinod1147** [DimitrisPapazafeiris/Website#1]
everything is good but your image should be round it would look great

**Comment by DimitrisPapazafeiris** [comment]
Yeah, i wanted to have it round but it had to be square and i didn't had one available
